biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -1073,6 +1075,8 @@ subroutine spawn_patches( currentSite, bc_in)
                                           currentSite%flux_diags%elem(el)%burned_liveveg + & 
                                           leaf_burn_frac * leaf_m * nc%n * area_inv
                                      
+                                     bc_out%fire_closs_to_atm_si = bc_out%fire_closs_to_atm_si + &
```

Comment:
Since this is in an element loop (ie does C,N,P), I think we need to add an if statement here to only calculate this for the carbon element.

Reply:
or... just move it outside the element loop and re-query leaf-carbon